### DISCUSSION TITLE
Editor issue : href="mailto:" getting sanitized

### DISCUSSION THREAD

**Discussion by danis-arslanov** [OrchardCMS/OrchardCore#7034] [Q&A] [answered]
If I use in widget Footer  `<a href=“mailto:mail@mail.com”>Link<\a>` it removes attribute href after page saving.

**Comment by jtkech** [comment]
Maybe because it is sanitized, in the admin look at the content definition related to this widget, then, if it has e.g. an html field, go to its settings and uncheck the `sanitize` option

**Comment by hishamco** [comment]
> Maybe because it is sanitized

I though that too, but I don't think there's a security issue with mail to unless there's a script injected

**Comment by deanmarcussen** [comment, marked as answer]
`mailto` is sanitized by default

See here to adjust the sanitizer https://docs.orchardcore.net/en/dev/docs/reference/core/Sanitizer/#configuring-the-sanitizer (oh and the example includes mailto)

Or as @jtkech says disable sanitization.

It's not something we'll include by default I suspect - the point of using an external sanitizer is that we do not need to make decisions about what is safe and isn't safe.

**Comment by danis-arslanov** [comment]
Thank you very much! You've been very helpful

**Comment by smokinguns** [comment]
@jtkech would turning this into a radio button so the user is forced into selecting one or the other. you can then warn them if they say no.  I know what to put in to avoid security vulnerabilities, but never saw the option for sanitize when setting up the template
